src/lib.rs

```diff
@@ -821,6 +821,9 @@ impl Connection {
     }
 
     /// Flush a NATS connection by sending a `PING` protocol and waiting for the responding `PONG`.
+    /// Will fail with `TimedOut` if the server does not respond with in 60 seconds.
```

Comment:
Nothing should take 60 secs, even around the world for ping I would wait max 2-5 secs.. ;)

Reply:
Ok, I copied the go client. I'll change it to 3 seconds.

Reply:
lol yea, understand.

Do we unblock them in Go client on disconnect immediately or wait through reconnect attempts. If immediately let's do 5 secs and I will make the Go client that as well.

Maybe send an email to have all clients consistent.